liquibase-core/src/main/java/liquibase/database/core/PostgresDatabase.java

```diff
@@ -23,18 +20,18 @@
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.sql.Statement;
-import java.sql.Types;
 import java.util.*;
 
 /**
  * Encapsulates PostgreSQL database support.
  */
 public class PostgresDatabase extends AbstractJdbcDatabase {
-    public static final String PRODUCT_NAME = "PostgreSQL";
+    private static final Logger log = LogFactory.getInstance().getLog();
+    private static final String PRODUCT_NAME = "PostgreSQL";
 
-    private Set<String> systemTablesAndViews = new HashSet<String>();
```

Comment:
The liquibase code is compiled with java 6, so I don't think you can take out the right-side generics here and below

Reply:
I didn't know. I'll refactor it

Reply:
Refactored. I've a question.
Why liquebase still on Java 6 ?